src/components/screens/wallet/transactions/transactionsList.js

```diff
@@ -93,6 +93,7 @@ class TransactionsList extends React.Component {
           : (
             <EmptyState className={styles.emptyState}>
               <Illustration name="emptyWallet" />
+              <h3>{t('No transactions found.')}</h3>
```

Comment:
Not having a message in empty transactions was according to the design https://projects.invisionapp.com/d/main?origin=v7#/console/17570736/365248392/preview?scrollOffset=518 as there already is a message in the empty chart box.

should it really be changed?

Reply:
As mentioned, requested by the designer.

Reply:
Though I agree, it was inconsistent.